.eslintrc

```diff
@@ -11,3 +11,6 @@ rules:
   yoda: 0
   arrow-parens: [2, "as-needed"]
   arrow-spacing: 2
+  dot-location: [2, "property"]
+  "max-len": [2, 80, 4, {"ignoreComments": true, "ignoreUrls": true, "ignorePattern": "^\\s*var\\s.+=\\s*require\\s*\\("}]
```

Comment:
@broucz ha, you are right :)
Fixed, I also simplified that rule a bit, so it has less than 80 chars long :)